src/Language/PureScript/CodeGen/JS/Printer.hs

```diff
@@ -121,7 +121,7 @@ literals = mkPattern' match'
     , prettyPrintJS' js
     ]
   match (Pure _ js) = mconcat <$> sequence 
-    [ return $ emit  "/* @__PURE__ */("
+    [ return $ emit  "/* #__PURE__ */("
```

Comment:
I'm curious about this change, the esbuild docs says it's fine with both. Do you know of any tools that don't work with `@` (or the other way around?)

Reply:
Yes I know, but both webpack and parcel use # in the documentation. So I changed it to this and since esbuild accepts both, I think it is better this way.